includes/amp-helper-functions.php

```diff
@@ -219,17 +231,6 @@ function() {
 function amp_after_setup_theme() {
 	amp_get_slug(); // Ensure AMP_QUERY_VAR is set.
 
-	/**
-	 * Filters whether AMP is enabled on the current site.
-	 *
-	 * Useful if the plugin is network activated and you want to turn it off on select sites.
-	 *
-	 * @since 0.2
-	 */
-	if ( false === apply_filters( 'amp_is_enabled', true ) ) {
-		return;
-	}
-
```

Comment:
Actually, this should be retained and if it is filtered to `false` we can add a `_doing_it_wrong()`.

Reply:
Done in 64b58db50